huaweicloud/resource_huaweicloud_as_group_v1.go

```diff
@@ -517,7 +527,34 @@ func resourceASGroupRead(d *schema.ResourceData, meta interface{}) error {
 
 	// set properties based on the read info
 	d.Set("scaling_group_name", asg.Name)
+	d.Set("vpc_id", asg.VpcID)
 	d.Set("status", asg.Status)
+
+	if asg.Status == "INSERVICE" {
+		d.Set("enable", true)
+	}
+	if asg.Status == "PAUSED" {
+		d.Set("enable", false)
+	}
```

Comment:
suggest to set `enable`  to false when the status not equal **INSERVICE**

Reply:
fixed